src/cuda_runtime_library_manager.h

```diff
@@ -39,6 +39,10 @@ class CUDARuntimeLibraryManager {
 
   ~CUDARuntimeLibraryManager();
 
+  using cudaError_t = ::cudaError_t;
+  using cudaIpcMemHandle_t = ::cudaIpcMemHandle_t;
+  using cudaMemcpyKind = ::cudaMemcpyKind;
+
```

Comment:
Further encapsulate types from `<cuda_runtime_api.h>` so external code doesn't need to include those headers.

There's probably cleaner/better ways to do this, so open to suggestions.